scripts/build-helpers.ps1

```diff
@@ -0,0 +1,307 @@
+#Requires -Version 5
+<#
+.SYNOPSIS
+    Common build script helper functions.
+    Intended to be dot sourced into the build script.
+
+.DESCRIPTION
+    Author: Blake Anderton, Joshua Poehls
```

Comment:
Copy/Pasta, might as well update it :)

Can you remove all the stuff you don't need from this file please.